### PR TITLE
DOC-627: Clarify existing table requirements

### PR DESCRIPTION
**Pull request by diana-macrometa**
Added a note to clarify existing table requirements for stream worker stores.

### CODE REVIEW

File: docs/cep/table/index.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ title: Tables (Collections)
 
 A _table_ is a stored version of an stream or a table of events. Its schema is defined in the _table definition_ that is similar to a stream definition. These events are stored in database. In Macrometa GDN, tables are called [collections](../../collections/).
 
+:::note
+If you want to use an existing Macrometa collection as a store, then you still need to define it in the stream worker and you must enable streaming on the collection.
```
Comment (StoyanTc) [inline docs/cep/table/index.md]: @diana-macrometa - The store may not have associated stream. This is not mandatory in this case.

Comment (diana-macrometa) [reply docs/cep/table/index.md]: I heard from Wayne that he couldn't get the existing collection to work until he turned the streaming on.

Comment (StoyanTc) [reply docs/cep/table/index.md]: Yes, this is true for a source with type='database'  but not for a table/store.

File: docs/cep/table/index.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ title: Tables (Collections)
 
 A _table_ is a stored version of an stream or a table of events. Its schema is defined in the _table definition_ that is similar to a stream definition. These events are stored in database. In Macrometa GDN, tables are called [collections](../../collections/).
 
+:::note
+If you want to use an existing Macrometa collection as a store, then you still need to define it in the stream worker.
```
Comment (jamesm-macrometa) [inline docs/cep/table/index.md]:
```suggestion
If you want to use an existing Macrometa collection as a store, then you must define it in the stream worker.
```

Your call on which wording you prefer

**Review by jamesm-macrometa — APPROVED** [review]

**Comment by diana-macrometa** [comment]
Heya @StoyanTc, will you be able to get to this one soon?

**Review by StoyanTc — APPROVED** [review]

**Pull request merged**